### PR TITLE
GUVNOR-3103: BindingTextBox extracted into shared package

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/widget/BindingTextBox.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2012 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/widget/BindingTextBox.java]: Is it worth updating the license.. IDK!?

Comment (jomarko) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/widget/BindingTextBox.java]: Neither I know. However, amended.

**Comment by jomarko** [comment]
@karreiro fixed.

**Comment by ederign** [comment]
@manstis could you please take a look on this?

**Comment by manstis** [comment]
@ederign This cannot be merged until https://github.com/kiegroup/drools-wb/pull/505 is fixed.

**Pull request merged**